src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -177,7 +177,8 @@ public function handle_update_order( WP_REST_Request $request ) {
 			return new WP_Error( 'tec-tc-gateway-paypal-nonexistent-order-id', null, $order );
 		}
 
-		$paypal_capture_response = tribe( Client::class )->capture_order( $paypal_order_id );
+		$payer_id = $request->get_param( 'payerID' ) ? $request->get_param( 'payerID' ) : '';
```

Comment:
Our side of the code shouldn't use `payerID` unless receiving the request directly from PayPal, which isn't the case here. We should snakecase it: `payer_id`.